### PR TITLE
[SYM-4245] Add prompt_field.prefetch to schema

### PR DESCRIPTION
**Pull request by Arianna2028**
Blocked by https://github.com/symopsio/platform/pull/1316

This PR adds a new `prefetch` attribute to `sym_flow.params.prompt_field`.

Test: because it wasn't reading `prefetch` before, the first apply after this change will show its value as `false` after the read from API. It will apply the change, and no weird diffs after that:
Screen Shot 2022-10-26 at 3 38 28 PM
Screen Shot 2022-10-26 at 3 41 10 PM

### CODE REVIEW

**Review by llam15 — APPROVED** [review]

File: sym/provider/flow_resource.go [hunk 1]
```diff
@@ -276,15 +277,46 @@ func readFlow(_ context.Context, data *schema.ResourceData, meta interface{}) di
...
+
+	for paramKey, paramValue := range flow.Params {
+		if paramKey == "prompt_fields" {
+			// Do the same check for known parameters within each prompt field
+			var knownPromptFields []map[string]interface{}
+
+			// Iterate over each prompt field from the API response
+			for i := range paramValue.([]interface{}) {
+				promptField := paramValue.([]interface{})[i].(map[string]interface{})
+				knownPromptField := map[string]interface{}{}
```
Comment (sym-andrewa) [inline sym/provider/flow_resource.go:294]: what the hell is this

`map[string]interface{}{}`

is this real?

**Review by sym-andrewa — COMMENTED** [review]
I tried reviewing this, but I don't understand Go well enough. 😅

File: sym/provider/flow_resource.go (hunk 1 above)
Comment (Arianna2028) [reply sym/provider/flow_resource.go:294]: hahahaha. this is actually fairly standard in go land.

`map[string]interface{}` is a type that means "map of string to any", basically.
and then you initialize it as an actual object with `{}` so `map[string]interface{}{}` is initializing a map of string to any

**Review by sym-andrewa — APPROVED** [review]
Thank you Ari for walking me through this lol

**Pull request merged**